src/test_validate.py

```diff
@@ -0,0 +1,53 @@
+from validate import parse_accept_language
```

Comment:
Nitpick: rename this to `validate_test.py` so it's always adjacent to the file it's testing when in sorted order.

Reply:
Cool, done!